package.json

```diff
@@ -1,14 +1,15 @@
 {
   "scripts": {
     "prepare": "husky install",
+    "jetify": "jetify",
     "start": "react-native start",
     "android:mosip": "react-native run-android --variant=mosipDebug",
     "ios": "react-native run-ios",
     "build:android:mosip": "cd android && ./gradlew :app:assembleMosipRelease && cd ..",
     "i18n:compile-strings": "node scripts/compile-strings.js",
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx --fix",
     "test": "jest",
-    "postinstall": "patch-package"
+    "postinstall": "patch-package && yarn jetify && sh tools/talisman/talisman-precommit.sh"
```

Comment:
can we use npm instead of yarn as we use npm by default

Reply:
Done replaced with npm run jetify